lib/metadata_json_lint.rb

```diff
@@ -5,11 +5,7 @@
 
 module MetadataJsonLint
   def options
-    @options ||= {
-      :fail_on_warnings    => true,
-      :strict_license      => true,
-      :strict_dependencies => false
-    }
+    @options ||= Struct.new(:fail_on_warnings, :strict_license, :strict_dependencies).new(true, true, false)
```

Comment:
I've reformatted it a bit. There is not much I can do about the way we need to create the struct. I was planning on using openstruct which takes a hash with defaults to create a new one but that opens up to people calling options.anything = without errors.